packages/cli/src/commands/org/search/dump.ts

```diff
@@ -81,18 +89,25 @@ export default class Dump extends Command {
     }),
   };
 
+  private aggregatedResults: RawResult[] = [];
+  private aggregatedFieldsWithDupes: string[] = [];
+  private temporaryDumpDirectory = dirSync();
```

Comment:
Would it be better that the temp directory is generated when `run()` is executed ?

Reply:
Good call, this actually cause the code to generate a temp folder at each command run, even others than dump. Oups.